src/core/migrations/0125_auto_20160505_1133.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+
+from core import models as core_models
+
+def load_setting(apps, schema_editor):
+    # We can't import the Person model directly as it may be a newer
+    # version than this migration expects. We use the historical version.
+    setting_group, created = core_models.SettingGroup.objects.get_or_create(name='email', enabled=True)
```

Comment:
In `RunPython` migrations, we shouldn't refer to model classes directly as they can change over time and break migrations.

Instead, we can use `apps.get_model`. You should be able to do something like:

    class = apps.get_model('core', 'SettingGroup') # Returns core_model.SettingGroup as it was defined when you wrote this migration
    setting_group, created = class.objects.get_or_create(…)

https://docs.djangoproject.com/en/1.10/ref/migration-operations/#django.db.migrations.operations.RunPython